redash/handlers/alerts.py

```diff
@@ -14,6 +14,12 @@ class AlertResource(BaseResource):
     def get(self, alert_id):
         alert = get_object_or_404(models.Alert.get_by_id_and_org, alert_id, self.current_org)
         require_access(alert.groups, self.current_user, view_only)
+        self.record_event({
+            'action': 'view',
+            'timestamp': int(time.time()),
```

Comment:
`timestamp` is being added by `self.record_event`,so no need to pass it manually (of course this applies in all other similar instances).